src/app/vault/ciphers.component.html

```diff
@@ -12,20 +12,22 @@
             (scrolled)="loadMore()">
             <a *ngFor="let c of filteredCiphers" appStopClick (click)="selectCipher(c)"
                 (contextmenu)="rightClickCipher(c)" href="#" title="{{'viewItem' | i18n}}"
-                [ngClass]="{'active': c.id === activeCipherId}">
+                [ngClass]="{'active': c.id === activeCipherId}" class="flex-list-item">
                 <app-vault-icon [cipher]="c"></app-vault-icon>
-                <span class="text">
-                    {{c.name}}
-                    <ng-container *ngIf="c.organizationId">
-                        <i class="fa fa-share-alt text-muted" title="{{'shared' | i18n}}" aria-hidden="true"></i>
-                        <span class="sr-only">{{'shared' | i18n}}</span>
-                    </ng-container>
-                    <ng-container *ngIf="c.hasAttachments">
-                        <i class="fa fa-paperclip text-muted" title="{{'attachments' | i18n}}" aria-hidden="true"></i>
-                        <span class="sr-only">{{'attachments' | i18n}}</span>
-                    </ng-container>
-                </span>
-                <span class="detail">{{c.subTitle}}</span>
+                <a class='flex-cipher-list-item'>
```

Comment:
Please use double quotes to encapsulate HTML attributes.

Reply:
Also, does the `a > a` structure mess with the tab index or keyboard interaction (accessibility)? Just wondering if it wouldn't be more semantically correct to use a `<span>` or since HTML5 allows it, a `<div>`? No particular prejudice against `<a>` here, just want to ensure it's interactive via keyboard still.

Reply:
Thanks and I see your point. I would probably opt for the `<div>` since `<span>` has a history of `display:inline`. Keyboard interaction looks to remain intact and accessibility is functioning the same as before as well.
Any thoughts around that before I push up these updates?

Thanks again for the feedback.

Reply:
Let's go with `<div>`, thanks!